replacy/__init__.py

```diff
@@ -2,12 +2,11 @@
 import itertools
 import logging
 import warnings
-from types import ModuleType
-from typing import Any, Callable, Dict, List, Optional, Tuple
-
 from functional import seq
 from spacy.matcher import Matcher
 from spacy.tokens import Span
+from types import ModuleType
+from typing import Any, Callable, Dict, List, Optional, Tuple
```

Comment:
this is a nit, but import order should be:
```python
import FROM_STD_LIB
from STD_LIB import WHATEVER

import EXTERNAL_DEP
from EXTERNAL_DEP import WHATEVER

from THIS_PACKAGE import WHATEVER
```

so this change makes it less correct

Reply:
I usually ran IDE optimize import and format code functions for every file I change, this is how I ended up with this change